applications/nrf_desktop/configuration/nrf52840_pca10059/ble_discovery.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/* This configuration file defines discovery parameters for BLE central.
+ * Used only by ble_discovery module.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} ble_discovery_include_once;
+
+#include "ble_event.h"
+
+struct bt_peripheral {
+	u16_t pid;
+	u8_t peer_type;
+};
+
+const u16_t vendor_vid = 0x1915;
+
+/* Peripherals parameters. */
+const struct bt_peripheral bt_peripherals[] = {
+	{
+		.pid = 0x52de,
```

Comment:
btw - we need to assign a different pids for red and black desktop mice

Reply:
Ok, I will make task for it